### PR TITLE
fix SetGridField

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-30932)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/SetGridField/SetGridField.py [hunk 1]
```diff
@@ -389,13 +390,13 @@ def main():  # pragma: no cover
...
         res = demisto.executeCommand("getIncidents", {"id": demisto.incident().get("id")})
-        is_playground = demisto.incident().get("isPlayground")
         custom_fields = {}
         for entry in res:
             if entry['Contents']:
                 data = entry["Contents"]["data"]
                 custom_fields = data[0].get("CustomFields") if data and data[0].get("CustomFields") else {}
-        if (not is_playground) and table and grid_id not in custom_fields:
+        # in the debugger and in the playground, there is an addition of the "_grid" suffix to the grid_id.
+        if is_xsiam_or_xsoar_saas() and table and (not (grid_id in custom_fields or f"{grid_id}_grid" in custom_fields)):
```
Comment (GuyAfik) [inline Packs/CommonScripts/Scripts/SetGridField/SetGridField.py:399]: can we simplify this condition in some way?

File: Packs/CommonScripts/ReleaseNotes/1_13_9.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### SetGridField
+
+- Fixed an issue where an error was raised when trying to run the script form the playbook debugger.
```
Comment (GuyAfik) [inline Packs/CommonScripts/ReleaseNotes/1_13_9.md:6]:
```suggestion
- Fixed an issue where an error was raised when trying to run the script from the playbook debugger.
```

**Review by GuyAfik — COMMENTED** [review]
LGTM, nice work

**Review by eyalpalo — APPROVED** [review]
Nice!

**Pull request merged**